cmd/zetae2e/local/zeta.go

```diff
@@ -59,6 +59,7 @@ func zetaTestRoutine(
 		// run zeta test
 		if err := zetaRunner.RunE2ETestsFromNames(
 			e2etests.AllE2ETests,
+			e2etests.TestZetaDepositName,
```

Comment:
```suggestion
```
Not necessary

Reply:
https://github.com/zeta-chain/node/pull/1791/commits/ce021e113520dbab64e1d27e5512c968b2dcefff